lib/cisco_node_utils/cmd_ref/vrf.yaml

```diff
@@ -18,9 +17,37 @@ description:
 destroy:
   config_set: "no vrf context <vrf>"
 
+feature_vn_segment_vlan_based:
+  _exclude:
+    - /N7K/  # Support for multi-tenancy full only
```

Comment:
I just tried your code on an n3132 and an n3048 and it fails because there's no 'vni' cli. Can you check with the vxlan guys and verify that vni is supported on n3k's? Also, point your minitest at their N7k and verify that this test works there as well.

```
  1) Error:
TestVrf#test_vrf_vni:
RuntimeError: [vrf test_vrf_vni] ' vni 4096' : % Invalid command
```